test/e2e/tests/settings/clear-activity.spec.js

```diff
@@ -41,6 +41,14 @@ describe('Clear account activity', function () {
         await driver.clickElement(
           '[data-testid="account-options-menu-button"]',
         );
+
+        // fix race condition with mmi build
```

Comment:
Nit: I was thinking about extracting this condition to a separate function, and when we convert this to POM e2e tests it would be super handy to transform too. wdyt>?

Reply:
thank you for your great suggestion @DDDDDanica  :bowing_woman: thanks to that I've realized that we already had the `openMenuSafe` function with the race condition check, so I'm using this now in the specs
(until POM is implemented, which will already check that in the page level --see changes in the pom file)

Reply:
That's great to hear ! nice solution !